package.json

```diff
@@ -19,6 +19,21 @@
   ],
   "main": "./dist/extension.js",
   "contributes": {
+    "configuration": {
+      "title": "env0",
+      "properties": {
+        "env0.apiUrl": {
+          "type": "string",
+          "required": true,
```

Comment:
`required` and `default` together? shouldn't it be one of them?
I think `required` is redundant here no?

Reply:
54080f8bf8af1a9c1c1e8607d3ba90c164fb37a6